hvpy/io.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+from pydantic import BaseModel
+
+
+class HvpyParameters(BaseModel):
+    def dict(self):
+        d = super().dict()
+        # Pydantic doesn't allow using lower case 'json' as a field,
+        # so override it here.
+        if "Json" in d:
+            d["json"] = d["Json"]
+            del d["Json"]
+        return d
+
+    def get_output_type(self):
+        # For now just return the raw output
+        return OutputType.Raw
+
+
+class OutputType(Enum):
+    # temporary strings, untill meeting with the team
+    Raw = "binary"
+    String = "string"
+    Json = "json"
```

Comment:
Do we want these to = a number instead of a string? I feel like the actual value is unimportant?

Reply:
The actual value isn't important as long as the enum is used consistently.
This is one of those Python things since enums aren't real enums

Reply:
Which is why I feel that numbers or even `auto()` might be neater.

Reply:
`auto()` looks like a neat choice.